locales/fr.lua

```diff
@@ -0,0 +1,20 @@
+local Translations = {
+    error = {
+        invalid_ext = "Ce n'est pas une extension valide, seul les liens utilisant les extensions %{fileext} sont autorisées. ",
```

Comment:
```suggestion
        invalid_ext = "Ce n'est pas une extension valide, seul les liens utilisant les extensions %{fileext} sont autorisées.",
```
Remove trailing space.